src/InstructionsForm.js

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+import React from 'react';
+
+export default function InstructionsForm({ instructions, setInstructions }) {
+  const [instructionInForm, setInstructionInForm] = useState('');
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    setInstructions([...instructions, instructionInForm]);
+  }
```

Comment:
if you add `setInstructionsForm('')` here, it will clear out the input field on submit